man/org.freedesktop.login1.xml

```diff
@@ -1268,6 +1271,11 @@ node /org/freedesktop/login1/session/1 {
       connection. This should help prevent a session from entering an inconsistent state, for example if the
       controller crashes. The only argument <varname>type</varname> is the new session type.</para>
 
+      <para><function>SetDisplay()</function> allows to change the display name of a graphical session. This is useful
```

Comment:
this seems useful indeed.

Reply:
> allows to change the display name of a graphical session

allows the display name of the graphical session to be changed